app/components/elements/Pagination.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import map from 'lodash/map';
+import remove from 'lodash/remove';
+import includes from 'lodash/includes';
+import capitalize from 'lodash/capitalize';
+import { Box, Text, BoxProps } from 'rebass/styled-components';
+import FirstPageRoundedIcon from '@material-ui/icons/FirstPageRounded';
+import LastPageRoundedIcon from '@material-ui/icons/LastPageRounded';
+import NavigateBeforeRoundedIcon from '@material-ui/icons/NavigateBeforeRounded';
+import NavigateNextRoundedIcon from '@material-ui/icons/NavigateNextRounded';
+import MoreHorizRoundedIcon from '@material-ui/icons/MoreHorizRounded';
+import { usePagination, PaginationProps } from '@material-ui/lab/Pagination';
+
+import Button from './Button';
+
+export const Pagination = props => {
+  const { id, variant, themeProps, ...paginationProps } = props;
+
+  const classNames = cx({
+    condensed: variant === 'condensed',
+  });
+
+  const { items } = usePagination(paginationProps);
+  const prevControls = remove(items, ({ type }) => includes(['first', 'previous'], type));
+  const nextControls = remove(items, ({ type }) => includes(['next', 'last'], type));
+
+  return (
+    <Box as="nav" variant={`paginators.${variant}`} {...themeProps}>
+      <ul className={classNames}>
+        <li>
+          <ul className="prev-controls">
+            {map(prevControls, ({ type, ...item }) => (
+              <li id={`${id}-${type}`} key={`${id}-${type}`}>
+                <Button px={2} variant="pagination" {...item}>
+                  {type === 'first' && <FirstPageRoundedIcon />}
+                  {type === 'previous' && <NavigateBeforeRoundedIcon />}
+                  {variant === 'default' && <Text pl={1}>{capitalize(type)}</Text>}
+                </Button>
+              </li>
+            ))}
+          </ul>
+        </li>
+        <li>
+          <ul className="pages">
+            {map(items, ({ page, type, selected, ...item }) => {
+              const pageClassNames = cx({
+                selected,
+              });
+
+              const itemId = type === 'page' ? `${id}-${page}` : `${id}-${type}`;
+              let children;
+
+              if (type === 'start-ellipsis' || type === 'end-ellipsis') {
+                children = <Button disabled className="ellipsis" variant="pagination"><MoreHorizRoundedIcon /></Button>;
```

Comment:
The a11y check in Storybook isn't terribly happy with this not having any text: `Element does not have inner text that is visible to screen readers`

Reply:
Good catch 👍 
I'll address this before merging.

Reply:
Fixed.  Now leveraging our `Icon` component which enforces the aria attributes.